rust/azure_iot_operations_services/src/azure_device_registry.rs

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+//! Types for Azure Device Registry operations.
+// use crate::azure_device_registry::device_name_gen::adr_base_service::client as adr_base_service_gen;
```

Comment:
Not sure if you meant to keep this  for this PR
```suggestion
```

Reply:
yes i did..did not want to cause issue in the lib.rs

Reply:
This is a comment though, how is it causing an issue?